### ISSUE TITLE
Change MediaCMS language

### ISSUE THREAD

**Issue by miltuss** [mediacms-io/mediacms#1017]
Is it possible to change the MediaCMS language?
 I would like to put the language in French

**Comment by dpf2000** [comment]
I am also very interested in this topic, as I would like to translate it into Brazilian Portuguese.

**Comment by azataiot** [comment]
Are you referring to the React interface or the Django backend ?

**Comment by cdorsat** [comment]
Can multi-language functions be added? The specific language and translation work to be added is left to the user.

**Comment by azataiot** [comment]
> Are you referring to the React interface or the Django backend ?

Are you referring to the React interface or the Django backend ?

**Comment by cdorsat** [comment]
> > Are you referring to the React interface or the Django backend ?
> 
> Are you referring to the React interface or the Django backend ?



To be precise, it should be the front end. What I mean mainly refers to the web display interface, that is, the website has the characteristics of multiple languages. If the administrator's setting interface also has multi-language capabilities, it will be more convenient for the administrator to manage the website.

**Comment by azataiot** [comment]
> > > Are you referring to the React interface or the Django backend ?
> > 
> > 
> > Are you referring to the React interface or the Django backend ?
> 
> To be precise, it should be the front end. What I mean mainly refers to the web display interface, that is, the website has the characteristics of multiple languages. If the administrator's setting interface also has multi-language capabilities, it will be more convenient for the administrator to manage the website.

If you are saying about the React side ( as you mentioned the UI ) then it should be the developers responsiblity of adding that capabillity. Search for i18n.

**Comment by cdorsat** [comment]
I don't know the programming skills in this area, so I don't know how to describe it accurately. In layman's terms, it is to add multi-language functions to the website, but the programming technicians are not responsible for the specific translation work, and the translation work can be left to the users.

**Comment by cdorsat** [comment]
In addition, I am willing to contribute to this project, but I do not know programming technology, but I can provide assistance in testing and make suggestions for improvements.

**Comment by mgogoulos** [comment]
They are now included in the latest release! Check it out on https://demo.mediacms.io/ and update!!!

Also if you want to help improving them, checkout https://github.com/mediacms-io/mediacms/blob/main/docs/admins_docs.md#20-translations

**Comment by miltuss** [comment]
@mgogoulos Thank you very much for your work, MediaCMS finally supports the French language thanks to you. it's really the best alternative to YouTube 👍
